lua/null-ls/builtins/diagnostics/deno_lint.lua

```diff
@@ -0,0 +1,58 @@
+local h = require("null-ls.helpers")
+local methods = require("null-ls.methods")
+local u = require("null-ls.utils")
+
+local DIAGNOSTICS = methods.internal.DIAGNOSTICS
+
+return h.make_builtin({
+    name = "deno_lint",
+    meta = {
+        url = "https://github.com/denoland/deno_lint",
+        description = "Blazing fast linter for JavaScript and TypeScript written in Rust",
+    },
+    method = DIAGNOSTICS,
+    filetypes = { "javaScript", "typescript", "typescriptreact", "javascriptreact" },
```

Comment:
```suggestion
    filetypes = { "javascript", "typescript", "typescriptreact", "javascriptreact" },
```

Actually not sure if these are case sensitive, but...